### ISSUE TITLE
Outbid button doesn't work sometimes

### ISSUE THREAD

**Issue by infidel04** [gergelyszabo94/csgo-trader-extension#436]
On the page https://steamcommunity.com/market/

Clicking the green outbid button sometimes just sets the price of a buy offer to be the same as the current highest bid. 

The buy order price goes from red to green but it doesn't outbid it. 

Happens like 10% of the time.  Has been happening for like the past 6 months.

**Comment by gergelyszabo94** [comment]
@infidel04 Have you noticed any pattern to which items ite happens to? Could it be that someone made the same bid between the extension checking the current highest bid and you clicking oudbid?

**Comment by infidel04** [comment]
I have not noticed a pattern.
No, that cant be the issue. It happens as often with very low volume Rust items.

**Comment by gergelyszabo94** [comment]
Okay, how expensive are these items btw?

**Comment by infidel04** [comment]
They vary a lot and I haven't noticed a pattern in price either. 
From 3-230 eur.

**Comment by infidel04** [comment]
idk if u did anything but it seems to be fixed. could've also just been my error all this time

**Comment by gergelyszabo94** [comment]
I haven't even tested it yet, no. I have not used this feature myself for a long time but I suspect I would have noticed it happening if it hapens 10% of the time. Something might have been changed since though,

**Comment by infidel04** [comment]
Hey, I have noticed it happening again, it really is a thing. Happens like 10-20% of the time for me.

The "outbid" button either matches the price of the highest offer or doesn't change the price at all.

**Comment by gergelyszabo94** [comment]
So I went on to investigate this now.
I set 10 buy orders for csgo items of varying prices. 
My buy orders were never the highest so when I loaded the market page they were all red:
![image](https://user-images.githubusercontent.com/3636827/177574223-f6265ace-dc8c-48d4-b785-208734aca25c.png)
Then I went on to outbid each of them, which worked as designed in all cases:
![image](https://user-images.githubusercontent.com/3636827/177574398-fa251094-2d0d-44d5-888c-3f3b5365d083.png)
I also opened each listing page after outbidding and I noticed that I was immediatelly outbid on one of the item. Reloading the market page confirmed this:
![image](https://user-images.githubusercontent.com/3636827/177574885-04cfb5ec-4124-4cc7-bc62-e58a3d02b88f.png)

In short I was not able to reproduce the issue.
It could be something language, currency or browser specific.
I am closing the issue now, feel free to reopen it if you find a way to consistently reproduce it.
